tests/Katana.Sandbox.WebServer/Startup.cs

```diff
@@ -132,7 +132,26 @@ public void Configuration(IAppBuilder app)
                 Authority = Environment.GetEnvironmentVariable("oidc:authority"),
                 ClientId = Environment.GetEnvironmentVariable("oidc:clientid"),
```

Comment:
This doesn't work unless you add the secret, right?
`ClientSecret = Environment.GetEnvironmentVariable("oidc:clientsecret"),`